ci/pipelines/deploy-to-perf.yml

```diff
@@ -0,0 +1,231 @@
+definitions:
+  aws_assumed_role_creds: &aws_assumed_role_creds
+    AWS_ACCESS_KEY_ID: ((.:role.AWS_ACCESS_KEY_ID))
+    AWS_SECRET_ACCESS_KEY: ((.:role.AWS_SECRET_ACCESS_KEY))
+    AWS_SESSION_TOKEN: ((.:role.AWS_SESSION_TOKEN))
+
+  wait_for_deploy_params: &wait_for_deploy_params
+    <<: *aws_assumed_role_creds
+    APPLICATION_IMAGE_TAG: ((.:application_image_tag))
+    NGINX_IMAGE_TAG: ((.:nginx_image_tag))
+    TELEGRAF_IMAGE_TAG: ((.:telegraf_image_tag))
+    ENVIRONMENT: test-perf-1
+
+  deploy_params: &deploy_params
+    <<: *aws_assumed_role_creds
+    APPLICATION_IMAGE_TAG: ((.:application_image_tag))
+    NGINX_IMAGE_TAG: ((.:nginx_image_tag))
+    TELEGRAF_IMAGE_TAG: ((.:telegraf_image_tag))
+    ACCOUNT: test
+    ENVIRONMENT: test-perf-1
+
+  check_release_versions_params: &check_release_versions_params
+    <<: *aws_assumed_role_creds
+    AWS_REGION: "eu-west-1"
+    CLUSTER_NAME: "test-perf-1-fargate"
+    APPLICATION_IMAGE_TAG: ((.:application_image_tag))
+    TELEGRAF_IMAGE_TAG: ((.:telegraf_image_tag))
+    NGINX_IMAGE_TAG: ((.:nginx_image_tag))
+
+  aws_test_config: &aws_test_config
+    aws_access_key_id: ((readonly_access_key_id))
+    aws_secret_access_key: ((readonly_secret_access_key))
+    aws_session_token: ((readonly_session_token))
+    aws_role_arn: arn:aws:iam::((pay_aws_test_account_id)):role/concourse
+    aws_ecr_registry_id: "((pay_aws_test_account_id))"
+    aws_region: eu-west-1
+
+  put_start_slack_notification: &put_start_slack_notification
+    put: slack-notification
+    params:
+      channel: '#govuk-pay-announce'
+      icon_emoji: ":fargate:"
+      username: pay-concourse
+      text: "((.:start_snippet)) |
```

Comment:
The links in these snippets are still pointing to the non-existent `alpha_release-perf` git tag (see https://github.com/alphagov/pay-ci/blob/master/ci/tasks/create-notification-snippets.yml#L25). We. could probably just hardcode some text here, like we do for the [e2e-helpers pipeline notifications](https://github.com/alphagov/pay-ci/blob/master/ci/pipelines/e2e-helpers.yml#L377)?

Reply:
[Fixed](https://github.com/alphagov/pay-ci/pull/679/commits/c0d07cb54f20c168dfb5f032217c3e28aac5e169)